R/cancensus.R

```diff
@@ -166,6 +178,21 @@ list_datasets <- function() {
   }
 }
 
+#' Return Census variable names and labels as a tidy data frame
+#'
+#' @return
+#'
+#' A data frame with a column \code{variable} containing the truncated 
+#' variable name, and a column \code{label} describing it.
+#'
+#' @export
+cancensus.labels <-   if("census_labels" %in% names(attributes(dat))) {
```

Comment:
Am I crazy, or is this not a function? Or is GitHub just showing me a broken diff?